site/_posts/2017-01-30-keeping-up-with-the-conferences.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Last Week in ManageIQ: Keeping up with the conferences"
+author: Dávid Halász
+date: 2017-01-30
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi, I am David from the ManageIQ UI team. I will be your guide in this quick recap of what happened in the last week on our premises.
+
+## Featured
+During the weekend, we were promoting ManageIQ on [DevConf.cz](http://devfonf.cz), an amazing conference held in Brno, Czech Republic. Mooli Tayer from the container management gave a really interesting talk about [automating a datacenter using ManageIQ and Ansible](https://www.youtube.com/watch?v=oK8l_u_kHC8). To learn more, check out our [twitter feed](https://twitter.com/manageiq) and look for the [#DevConfCZ](https://twitter.com/search?l=&q=%23devconfcz%20from%3Amanageiq&src=typd) hashtag.
+
+Even if some of us were busy with organizing the conference, we were able to merge 124 PRs into 6 repositories. We are still getting used to the new splitted repositories, so there are plenty of PRs assuring that everything works well among these repos.
+
+## Improved
+
+[Eric Winchell](https://github.com/epwinchell) is responsible for having more and more vector graphics in the UI. His plan is to use fonticons where we don't need more than one color and SVG files where colors are necessary. His [last merged PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/249) gets rid of references to PNG files in trees related to Automation and Policies. Hopefully when we are fully converted across the whole UI, we can get rid a hundreds of unused PNG files.
+
+Ansible is great and we would like to have better integration with our automation. [James](https://github.com/ManageIQ/manageiq/pull/13630) is one of the people working on this very hard. By [migrating](https://github.com/ManageIQ/manageiq/pull/13630) the `ConfigurationManager` of Ansible Tower to the new `AutomationManager` we are one step closer in achieving this goal.
+
+## Fixed
+
+As we are trying to move the VMware provider to its new repo, some paths in requiring helpers weren't been adjusted properly. Thanks to [Adam](https://github.com/agrare) this is now [fixed and working](https://github.com/ManageIQ/manageiq-providers-vmware/pull/2). These things are expected consequences of the moving, but they can be found and fixed quickly.
```

Comment:
> some paths in requiring helpers weren't been adjusted properly

some paths in requiring helpers weren't being adjusted properly.

or just

some paths in requiring helpers weren't adjusted properly.